spec/_attachments/ic.did

```diff
@@ -121,6 +128,16 @@ type node_metrics = record {
     num_block_failures_total : nat64;
 };
 
+type log_record = record {
+  idx: nat;
```

Comment:
given the comment on timestamps, we could also consider using nat64 here for consistency

Reply:
Done.